dockerhub/Dockerfile

```diff
@@ -3,15 +3,22 @@ FROM node:12-alpine
 ENV REGION="US"
 
 RUN \
-  apk add --update bash tini && \
+  apk add --no-cache --update bash tini && \
   rm -rf /var/cache/apk/*
 
-RUN \
-  npm install -g --unsafe-perm @sematext/logagent && \
+RUN \ 
+  apk add --no-cache --virtual .build-deps alpine-sdk python && \ 
+  npm install -g --unsafe-perm --production git+https://sematext@github.com/sematext/logagent-js.git && \
+  npm install -g --unsafe-perm --production @sematext/logagent && \
+  npm install -g --production --unsafe-perm @sematext/logagent-nodejs-monitor && \
   mkdir -p /etc/sematext && \
+  mkdir -p /etc/logagent && \
```

Comment:
Do we need both `/etc/sematext` and `/etc/logagent` dirs?

Reply:
Yes. We might change it, but we have to check documentation, backward compatibility  etc. I think pattern files are expected in /etc/logagent and logagent.conf in /etc/sematext.